ui/src/App.js

```diff
@@ -67,6 +100,10 @@ function App() {
             <Route path="/documents">
               <Documents />
             </Route>
+            <Route path="/Home" exact>
```

Comment:
route name should be small letters  '/home'

Reply:
done